app/src/main/res/layout/main_fragment.xml

```diff
@@ -30,30 +30,75 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:padding="@dimen/space_normal">
+        android:layout_height="match_parent">
 
-        <Button
+        <FrameLayout
+            android:id="@+id/toolbar"
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:background="@color/brand_primary"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_gravity="center"
+                android:background="@color/brand_primary"
+                app:srcCompat="@drawable/ic_logo" />
+        </FrameLayout>
+
+        <View
+            android:id="@+id/headerBackgroundView"
+            android:layout_width="match_parent"
+            android:layout_height="144dp"
+            android:background="@color/brand_primary"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/toolbar" />
```

Comment:
I am not sure if we should think of this background as a part of the toolbar or separate. 🤔
You can double-check with @hekanaho or @ninnayaa so we know how they see it.